gazebo_ros/src/utils.cpp

```diff
@@ -43,4 +49,15 @@ std::string ScopedNameBase(const std::string & str)
   return str.substr(idx + 2);
 }
 
+std::string SensorFrameID(const gazebo::sensors::Sensor & _sensor, const sdf::Element & _sdf)
+{
+  // Return frame from sdf tag if present
+  if (_sdf.HasElement("frame_name")) {
```

Comment:
Will this tag usually be inside `<ros>`?

Reply:
I think we should reserve `<ros>` for configuring a `gazebo_ros::Node` specifically. As this is used in the imu PR, the tag appears in the root of the plugin's SDF.

Reply:
Cool, thanks for the clarification. Let merge this!